src/routes/dashboard/project/[project_id]/library/Library.svelte

```diff
@@ -68,6 +77,23 @@
 			query.textSearch('search_field', filter);
 		}
 
+		if (selected_projects.length > 0) {
+			if (selected_projects.find((p) => p === -1)) {
+				query.or(
+					`project_id.in.(${selected_projects.filter((p) => p !== -1).join(',')}),project_id.is.null`
+				);
+			} else {
+				query.in(
+					'project_id',
+					selected_projects.filter((p) => p !== -1)
+				);
+			}
```

Comment:
```suggestion
			if (selected_projects.find((p) => p === -1)) {
				query.or(
					`project_id.in.(${real_selected_projects.join(',')}),project_id.is.null`
				);
			} else {
				query.in(
					'project_id',
					real_selected_projects
				);
			}
```

avec la déclaration au-dessus pour plus de lisibilité 🙏